compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/Platform.skiko.kt

```diff
@@ -77,8 +76,12 @@ internal interface Platform {
             }
 
             override fun setPointerIcon(pointerIcon: PointerIcon) = Unit
-            override fun viewConfiguration(densityProvider: () -> Density) =
-                DefaultViewConfigurationWithDensityProvider(densityProvider)
+            override val viewConfiguration = object : ViewConfiguration {
+                override val longPressTimeoutMillis: Long = 0L
```

Comment:
Better to use non-zero values for the default implementation (but for touchSlop it is okay)